pkg/e2e/actuators/actuators.go

```diff
@@ -0,0 +1,441 @@
+package actuators
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/golang/glog"
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	e2e "github.com/openshift/cluster-api-actuator-pkg/pkg/e2e/framework"
+	mapiv1beta1 "github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	kpolicyapi "k8s.io/api/policy/v1beta1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/apimachinery/pkg/util/wait"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	workerRoleLabel = "node-role.kubernetes.io/worker"
```

Comment:
these values are duplicated in operator common. Can we have some common utils?

Reply:
nit: Anything but `common` or `utils`. Something with more explicit naming please.

Reply:
Yeah, I will create `pkg/e2e/common` package.